src/ucp/core/ucp_mm.h

```diff
@@ -50,7 +50,7 @@ enum {
 typedef struct ucp_rkey {
     /* cached values for the most recent endpoint configuration */
     struct {
-        ucp_ep_cfg_index_t        ep_cfg_index; /* EP configuration relevant for the cache */
+        ucp_worker_cfg_index_t    ep_cfg_index; /* EP configuration relevant for the cache */
```

Comment:
`worker_cfg_index` or just `cfg_index`?

Reply:
worker_cfg_index could help in case there is other configuration which is not on the worker (maybe on the context)

Reply:
i was just proposing a name change